apps/src/templates/rubrics/LearningGoal.jsx

```diff
@@ -0,0 +1,67 @@
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+import i18n from '@cdo/locale';
+import style from './rubrics.module.scss';
+import {learningGoalShape} from './rubricShapes';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+import {BodyThreeText} from '@cdo/apps/componentLibrary/typography';
+
+export default function LearningGoal({
+  learningGoal,
+  teacherHasEnabledAi,
+  canProvideFeedback,
+}) {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const aiEnabled = learningGoal.aiEnabled && teacherHasEnabledAi;
+
+  return (
+    <details
+      className={style.learningGoalRow}
+      onClick={() => setIsOpen(!isOpen)}
+    >
+      <summary className={style.learningGoalHeader}>
+        <div className={style.learningGoalHeaderLeftSide}>
+          {isOpen && (
+            <FontAwesome
+              icon="angle-up"
+              onClick={() => setIsOpen(false)}
+              className={style.arrowIcon}
+            />
+          )}
+          {!isOpen && (
+            <FontAwesome
+              icon="angle-down"
+              onClick={() => setIsOpen(true)}
+              className={style.arrowIcon}
+            />
+          )}
+          <span>{learningGoal.learningGoal}</span>
+        </div>
+        <div className={style.learningGoalHeaderRightSide}>
+          {aiEnabled && <AiToken />}
+          {/*TODO: Display status of feedback*/}
+          {canProvideFeedback && <BodyThreeText>Needs approval</BodyThreeText>}
+        </div>
+      </summary>
+      <div className={style.learningGoalExpanded}>
+        Learning Goal Evidence Levels will be here.
+      </div>
+    </details>
+  );
+}
+
+LearningGoal.propTypes = {
+  learningGoal: learningGoalShape.isRequired,
+  teacherHasEnabledAi: PropTypes.bool,
+  canProvideFeedback: PropTypes.bool,
+};
+
+const AiToken = () => {
```

Comment:
Just for my own understanding, why did you decide to create this function separately than above?  Was it a readability issue or do you think you will want to use this AiToken function elsewhere?

Reply:
Mostly readability. In particular, when things are conditionally shown, I personally prefer to pull them out if possible because I find

```
{aiEnabled && <AiToken />}
```

easier to read than

```
{aiEnabled && (
  <BodyThreeText className={style.aiToken}>
      {i18n.artificialIntelligenceAbbreviation()}
      <FontAwesome icon="check" />
   </BodyThreeText>
)}
```

Definitely just a personal preference though :)